pkg/controller/gather_commands.go

```diff
@@ -399,7 +420,7 @@ func wasDataProcessed(ctx context.Context,
 // If not, non-nill error is returned.
 func (g *GatherJob) storagePathExists() error {
 	if _, err := os.Stat(g.StoragePath); err != nil && os.IsNotExist(err) {
-		if err = os.MkdirAll(g.StoragePath, 0777); err != nil {
+		if err = os.MkdirAll(g.StoragePath, 0o777); err != nil {
```

Comment:
why is this required please? :thinking:

Reply:
It is the newer Go syntax to write the octal numbers that should be more readable and mine IDE updated this line. It is not required and we could stick with the old format if you prefer that.